### DISCUSSION TITLE
It's possible to extract the data to PDF ?

### DISCUSSION THREAD

**Discussion by ecatugy** [icflorescu/mantine-datatable#519] [General]
The component is really good, offering a great customization capability. I'd like to know if it's possible to extract the data to PDF, or if there's any possibility of this feature in a future release.

**Comment by icflorescu** [comment]
Hey, @ecatugy, this is definitely beyond the scope of this project.

PDF generation is not trivial, and certainly not something you could do in a couple of lines of code for a complex table
I have successfully used ReactPDF to build invoices and reports in my commercial projects: https://react-pdf.org/
[Diego Muracciole](https://github.com/diegomura) did an outstanding job with it, the documentation website is quite good, and if I remember correctly it works both on the server-side and on the client-side.

So I'd suggest you to give it a try.
